launch/sensor_mtp.launch

```diff
@@ -0,0 +1,70 @@
+<launch>
+
+  <arg name="ouster_ns" default="ouster" doc="Override the default namespace of all ouster nodes"/>
+  <arg name="sensor_hostname" doc="hostname or IP in dotted decimal form of the sensor"/>
+  <arg name="udp_dest" default=" " doc="hostname or multicast group IP where the sensor will send UDP data packets"/>
```

Comment:
If this can not be empty in a multicast setup then I would suggest that you remove the default attribute, thus making this argument a required argument for this launch file.
```suggestion
  <arg name="udp_dest" doc="hostname or multicast group IP where the sensor will send UDP data packets"/>
```